test/e2e/dns.go

```diff
@@ -41,6 +41,8 @@ var dnsServiceLabelSelector = labels.Set{
 	"kubernetes.io/cluster-service": "true",
 }.AsSelector()
 
+var ClusterDNSVerifier = verifyDNSPodIsRunning
```

Comment:
Does this mean we won't be able to change this var without breaking people outside of this repo? If so will you make a comment to this effect?